srcpkgs/grafana/INSTALL

```diff
@@ -4,6 +4,8 @@ case "$ACTION" in
             :
         else
             chown _grafana:_grafana var/lib/grafana
+            vmkdir var/log/grafana
```

Comment:
vmkdir is not available in an INSTALL script.  You need to do a normal mkdir here.